astropy/visualization/interval.py

```diff
@@ -6,39 +6,73 @@
 """
 
 from __future__ import division, print_function
-
```

Comment:
I do order them, just without blank lines. :)  Ah, OK.  `pep8` has failed me all these years (it could at least check the standard library imports).